WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabItemsStore.swift

```diff
@@ -86,14 +86,35 @@ extension ReaderTabItemsStore {
             return
         }
         state = .loading
-        service.fetchReaderMenu(success: { [weak self] in
-                                    self?.fetchTabBarItems()
-            },
-                                failure: { [weak self] error in
-                                    let actualError = error ?? NSError(domain: WordPressComRestApiErrorDomain, code: -1, userInfo: nil)
-                                    DDLogError(ReaderTopicsConstants.remoteFetchError + actualError.localizedDescription)
-                                    self?.fetchTabBarItems()
+
+        let dispatchGroup = DispatchGroup()
+
+        // Sync the reader menu
+        dispatchGroup.enter()
+        service.fetchReaderMenu(success: {
+            dispatchGroup.leave()
+        }, failure: { (error) in
+            let actualError = error ?? ReaderTopicsConstants.remoteServiceError
+            DDLogError(ReaderTopicsConstants.remoteFetchError + actualError.localizedDescription)
+
+            dispatchGroup.leave()
+        })
+
+        // Sync the followed sites
+        dispatchGroup.enter()
+        service.fetchFollowedSites(success: {
+            dispatchGroup.leave()
+        }, failure: { (error) in
+            let actualError = error ?? ReaderTopicsConstants.remoteServiceError
+            DDLogError(ReaderTopicsConstants.remoteFetchError + actualError.localizedDescription)
```

Comment:
Shouldn't this error logging be something like `DDLogError("Could not sync sites: \(String(describing: error))") ?

Reply:
The `ReaderTopicsConstants.remoteFetchError` maps to a string `Error syncing menu:`.

I updated the error logging to reflect the errors `ReaderMenuViewController` was using and switch to `String(describing:)`